package.json

```diff
@@ -44,6 +45,7 @@
     "test:coverage": "rm -rf ./coverage && NODE_ENV=coverage npm run karma",
     "test:cli": "node ./cli/test.js",
     "test:setup": "playwright install firefox webkit --with-deps",
+    "test:typescript": "npx tsc --noEmit",
```

Comment:
You can remove `npx` if the package is local. NPM runs commands from the local packages just by their names.

Please add the `test:typescript` commend to `test` - as for now it only runs when called manually (I would add it after the lint).

Reply:
Command is still not added to `test`.

Reply:
I'm not sure what response is expected from the launched process and I'm not familiar with how CI/CD works in a project. I don't think that `test:typescript` will return an error code if it finds errors in the code. I made a change but can't test it.